test/unit/bookmark_test.rb

```diff
@@ -20,6 +20,15 @@ class BookmarkTest < ActiveSupport::TestCase
     end
   end
 
+  test "my bookmarks should be able to create two bookmarks with same name under different controllers" do
+    enable_login do
```

Comment:
Your right, I've removed it.
Should i remove it for the rest of the tests on bookmark?